src/python/include/Simplex_tree_interface.h

```diff
@@ -53,7 +53,10 @@ class Simplex_tree_interface : public Simplex_tree<SimplexTreeOptions> {
   }
 
   void assign_simplex_filtration(const Simplex& simplex, Filtration_value filtration) {
-    Base::assign_filtration(Base::find(simplex), filtration);
+    Simplex_handle sh = Base::find(simplex);
+    if (sh == Base::null_simplex())
+      throw std::invalid_argument("Cannot assign a filtration to a simplex that is not in the complex");
```

Comment:
Should we explain that the python method will throw an exception in simplex is not found ? `ValueError` in python

Reply:
"Obvious" exceptions seem to be not documented under python cf.  [numpy.genfromtxt](https://numpy.org/devdocs/reference/generated/numpy.genfromtxt.html#numpy-genfromtxt), `np.genfromtxt("unexisting_file.txt")` throws a `FileNotFoundError`